docker/deploy/docker-compose.yml

```diff
@@ -37,24 +37,31 @@ services:
             - MYSQL_PASSWORD=mysecretpw
             - MYSQL_DATABASE=SQE
             
-            ## Email settings (account that mails user registration information)
+            ## Email settings (account that mails user registration information), these are required.
```

Comment:
You should probably take the docker-compose.yml file from the ScrollEditor repository, which takes MAILER_* variables from the environment.